gobblin-restli/gobblin-job-config-service/api/src/main/pegasus/gobblin/service/JobConfig.pdsc

```diff
@@ -0,0 +1,32 @@
+{
+  "type" : "record",
+  "name" : "JobConfig",
+  "namespace": "gobblin.service",
+  "doc" : "Configuration for a job",
+  "fields" : [
+    {
+      "name" : "jobName",
+      "type" : "string",
+      "doc" : "Name of the job"
+    },
+    {
+      "name" : "jobGroup",
+      "type" : "string",
+      "doc" : "Group of the job"
+    },
+    {
+      "name" : "schedule",
+      "type" : "string",
+      "doc" : "Schedule for job"
```

Comment:
Can you make it clear what the format of the schedule string is? (e.g. cron-style)